### ISSUE TITLE
Improving int8 quantization results. 

### ISSUE THREAD

**Issue by severecoder** [NVIDIA/TensorRT#3865]
I have used PTQ for int8 export from pytorch model and despite attempts at calibration, there is a significant drop in detection accuracy. 

I am moving to quantization aware training to improve the accuracy, to improve the quantized int8 model, is pytorch_quantization the best tool for that?

The end result is to have .trt or engine file inferencing at int8 precision with best possible detection metrics.

TIA

**Comment by zerollzeng** [comment]
> I am moving to quantization aware training to improve the accuracy, to improve the quantized int8 model, is pytorch_quantization the best tool for that?

pytorch_quantizaton will be deprecated, please use AMMO now.

**Comment by severecoder** [comment]
Thank for the response, isn't ammo only limited to LLMs?

**Comment by brb-nv** [comment]
There's also support for diffuser models. [[link](https://github.com/NVIDIA/TensorRT-Model-Optimizer)]

Btw, AMMO has been renamed to TensorRT Model Optimizer. [[reference](https://github.com/NVIDIA/TensorRT-LLM/issues/1368#issuecomment-2118356237)][](url)
